src/app/page/component/content-image/content-image.component.ts

```diff
@@ -60,6 +70,11 @@ export class ContentImageComponent implements OnChanges {
     this.width = dimensions?.value
       ? dimensions.value + dimensions.symbol
       : null;
+    const isEventType = !!this.image.events?.length;
+    if (isEventType) {
+      this.type = 'event';
+      this.events = this.image.events;
+    }
```

Comment:
```suggestion
    this.events = this.image.events
    this.isEventType = this.events?.length
```

Reply:
Then you cane just use `isEventType` in the HTML instead of comparing it to `= 'events'`.